export/main.ts

```diff
@@ -26,6 +29,12 @@ async function fetchProjectData(projectId: string, group?: string): Promise<any>
             console.log(await response.text());
         } else {
             const data = await response.json();
+            const latestJobId = data.latestJobId;
+
+            if(latestJobId) {
+                return evaluatePolicy(latestJobId, policy);
```

Comment:
This seems like you get back a different schema if there is a latest job ID returned.

Reply:
You're right. I cleaned this up to return a null value if no data is found. We only write out the data if we have at least one job now.